datacapture/src/test/java/com/google/android/fhir/datacapture/mapping/ResourceMapperTest.kt

```diff
@@ -46,14 +50,26 @@ import org.hl7.fhir.r4.model.StringType
 import org.hl7.fhir.r4.model.codesystems.AdministrativeGender
 import org.hl7.fhir.r4.utils.StructureMapUtilities
 import org.intellij.lang.annotations.Language
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.robolectric.RobolectricTestRunner
 import org.robolectric.annotation.Config
+import org.robolectric.util.ReflectionHelpers
 
 @RunWith(RobolectricTestRunner::class)
-@Config(sdk = [Build.VERSION_CODES.P])
+@Config(sdk = [Build.VERSION_CODES.P], application = DataCaptureTestApplication::class)
 class ResourceMapperTest {
+
+  @Before
+  fun setUp() {
+    check(
+      ApplicationProvider.getApplicationContext<DataCaptureTestApplication>() is
+        DataCaptureConfig.Provider
+    ) { "Few tests require a custom application class that implements DataCaptureConfig.Provider" }
+    ReflectionHelpers.setStaticField(DataCapture::class.java, "configuration", null)
+  }
+
   @Test
   fun `extract() should perform definition-based extraction`() {
```

Comment:
you can simplify this by:

```suggestion
  fun `extract() should perform definition-based extraction`() = runBlocking {
    ...
  }
```

Reply:
and the same for the other test cases

Reply:
Done for all the tests.